src/sectors.jl

```diff
@@ -220,6 +228,37 @@ Base.:&(::SimpleFusion, ::UniqueFusion) = SimpleFusion()
 Base.:&(::GenericFusion, ::UniqueFusion) = GenericFusion()
 Base.:&(::GenericFusion, ::SimpleFusion) = GenericFusion()
 
+# similar, but for multifusion categories
+"""
+    MultiFusionStyle(::Sector)
+    MultiFusionStyle(I::Type{<:Sector})
+
+Trait to describe the semisimplicity of the unit sector of type `I`.
+This can be either
+*   `SimpleMultiFusion()`: the unit is simple (e.g. fusion categories);
+*   `GenericMultiFusion()`: the unit is semisimple.
+"""
+abstract type MultiFusionStyle end
+MultiFusionStyle(a::Sector) = MultiFusionStyle(typeof(a))
+
+struct SimpleMultiFusion <: MultiFusionStyle end
+struct GenericMultiFusion <: MultiFusionStyle end
+
+MultiFusionStyle(::Type{<:Sector}) = SimpleMultiFusion() # default is simple unit
```

Comment:
We could also have a default implementation that depends only on correctly implementing `allones`:
```suggestion
MultiFusionStyle(::Type{I}) where {I<:Sector} = length(allones(I)) == 1 ? SimpleMultiFusion() : GenericMultiFusion()
```

Note however that this is incompatible with the suggestion above to throw a methoderror if the fusionstyle is not correct, since this would otherwise cause an infinite loop.